oracle/price-feeder/oracle/client/tx.go

```diff
@@ -25,15 +25,15 @@ func BroadcastTx(clientCtx client.Context, txf tx.Factory, msgs ...sdk.Msg) (*sd
 	// }
 
 	// txf = txf.WithGas(adjusted)
+	txf = txf.WithGas(0)
+	// txf = txf.WithFees("0usei")
 
 	unsignedTx, err := tx.BuildUnsignedTx(txf, msgs...)
 	if err != nil {
 		return nil, err
 	}
 
-	// unsignedTx.SetFeeGranter(clientCtx.GetFeeGranterAddress())
-	unsignedTx.SetFeeAmount(sdk.NewCoins(sdk.NewCoin("usei", sdk.NewInt(0))))
-	// unsignedTx.SetFeePayer(clientCtx.GetFeePayerAddress())
```

Comment:
how come we dont need to set the fee amount anymore?

Reply:
because we don't need to send any fees with the transaction (I believe this behaves the same as explicitly setting 0?)